### PR TITLE
[Event Hubs] migration guide update

### PR DESCRIPTION
**Pull request by YijunXieMS**

### CODE REVIEW

File: sdk/eventhub/azure-eventhub/migration_guide.md [hunk 1]
```diff
@@ -1,14 +1,17 @@
-# Migration Guide (EventHubs v1 to v5)
+# Guide to migrate from azure-eventhub v1 to v5
 
 This document is intended for users that are familiar with V1 of the Python SDK for Event Hubs library (`azure-eventhub 1.x.x`) and wish 
 to migrate their application to V5 of the same library.
 
 For users new to the Python SDK for Event Hubs, please see the [readme file for the azure-eventhub](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/eventhub/azure-eventhub/README.md).
 
 ## General changes
+Version 5 of the azure-eventhub package is a result of our efforts to create a client library that is user-friendly and idiomatic to the Python ecosystem.
```
Comment (annatisch) [inline sdk/eventhub/azure-eventhub/migration_guide.md]: ...is _the_ result...

File: sdk/eventhub/azure-eventhub/migration_guide.md [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
+Apart from redesigns resulting from the new [Azure SDK Design Guidelines for Python](https://azure.github.io/azure-sdk/python_introduction.html#design-principles),
```
Comment (annatisch) [inline sdk/eventhub/azure-eventhub/migration_guide.md]: "Alongside API redesigns driven by the new Azure SDK Design Guidelines, the latest version...."

File: sdk/eventhub/azure-eventhub/migration_guide.md [hunk 3]
```diff
@@ -1,14 +1,17 @@
...
 For users new to the Python SDK for Event Hubs, please see the [readme file for the azure-eventhub](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/eventhub/azure-eventhub/README.md).
 
 ## General changes
+Version 5 of the azure-eventhub package is a result of our efforts to create a client library that is user-friendly and idiomatic to the Python ecosystem.
+Apart from redesigns resulting from the new [Azure SDK Design Guidelines for Python](https://azure.github.io/azure-sdk/python_introduction.html#design-principles), 
+the latest version improves on several areas from V1.
 
-In the interest of simplifying the API surface we've made two distinct
-clients, rather than having a single `EventHubClient`. 
+### Specific clients for sending vs receiving
```
Comment (annatisch) [inline sdk/eventhub/azure-eventhub/migration_guide.md]: "sending and receiving"

File: sdk/eventhub/azure-eventhub/migration_guide.md [hunk 4]
```diff
@@ -31,13 +34,16 @@ The code samples in this migration guide use async APIs.
 | `EventHubClientAsync.from_connection_string()` | `EventHubProducerClient.from_connection_string` or `EventHubConsumerClient.from_connection_string` |[client creation](./samples/async_samples/client_creation_async.py) |
 | `EventProcessorHost()`| `EventHubConsumerClient(..., checkpoint_store)`| [receive events using checkpoint store](./samples/async_samples/recv_with_checkpoint_store_async.py) |
 
+In V5, the SDK provides `BlobCheckpointStore`
```
Comment (annatisch) [inline sdk/eventhub/azure-eventhub/migration_guide.md]: BlobCheckPointStore isn't in v5, it has been removed from v5 into a separate package

Comment (YijunXieMS) [reply sdk/eventhub/azure-eventhub/migration_guide.md]: Changed to "In V5, the SDK provides `BlobCheckpointStore` in extension packages azure-eventhub-checkpointstoreblob (for sync) and azure-eventhub-checkpointstoreblob-aio (for async).
You can define your own `CheckpointStore` class to persist checkpoint data."

File: sdk/eventhub/azure-eventhub/migration_guide.md (hunk 3 above)
Comment (YijunXieMS) [reply sdk/eventhub/azure-eventhub/migration_guide.md]: Updated

File: sdk/eventhub/azure-eventhub/migration_guide.md (hunk 2 above)
Comment (YijunXieMS) [reply sdk/eventhub/azure-eventhub/migration_guide.md]: Updated

File: sdk/eventhub/azure-eventhub/migration_guide.md (hunk 1 above)
Comment (YijunXieMS) [reply sdk/eventhub/azure-eventhub/migration_guide.md]: Updated

**Pull request merged**